HomeTask1/main.css

```diff
@@ -0,0 +1,197 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+    outline: 0;
+    font-size: 100%;
+    vertical-align: baseline;
+    font-family: 'Roboto', sans-serif;
+}
+
+:root{
+    --primarycolor:#273d54;
+    --whitecolor:rgb(248, 246, 246);
+    --buttoncolor:#c76451;
+    --graycolor:#707887;
+    --separator: #aba491;
+    --blackcolor:rgb(0, 0, 0);
+}
+
+header{
+    background-color: var(--primarycolor);
+    display: flex;
+    height: 6rem;
+}
+.titlebox{
+    padding: 1%;
+    margin:auto;
+    background-color: var(--whitecolor);
+    text-align: center;
+    border: 2px solid var(--buttoncolor);
+}
+
+h1{
+    font-size: x-large;
+}
+
+p{
+    font-size: x-small;
+}
+
+main{
+    margin-bottom: 20px;
+}
+
+section{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 80%;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    font-weight: bold;
+    border-bottom: 1.5px solid var(--graycolor);
+    margin: auto;
+    color:var(--primarycolor);
+}
+
+.select{
+    font-size: small;
+}
+#filter{
+    border:1px solid;
+    margin-top: 5px;
+    padding: 5px;
+}
+h2{
+    font-size: x-large;
+    font-weight: 700;
+
+}
+.scontainer{
+    /* background-color: red; */
```

Comment:
Remove commented code if unused. And also class

Reply:
Removed comments and unused code as requested.